### PR TITLE
Updating Role Resource

### PR DESCRIPTION
**Pull request by dino-saurabh**
1. list operations examples added

### CODE REVIEW

**Comment by dino-saurabh** [comment]
@Akayeshmantha  please review !!

File: source/includes/User/role.md [hunk 1]
```diff
@@ -14,76 +14,307 @@ A **Role** represents a group of privileges in the system. Roles may inherit pri
...
+  .addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
+  .addHeader("Cookie", "JSESSIONID=955E778A4F700C5AA9651DAF6FC9DDDA")
+  .build();
+Response response = client.newCall(request).execute();
+
+```
+
+```javascript
+
+var myHeaders = new Headers();
```
Comment (Akayeshmantha) [inline source/includes/User/role.md]: @LuGO0  can you change the name my_header to  a one sensible

File: source/includes/User/role.md [hunk 2]
```diff
@@ -14,76 +14,307 @@ A **Role** represents a group of privileges in the system. Roles may inherit pri
...
+myHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
+myHeaders.append("Cookie", "JSESSIONID=955E778A4F700C5AA9651DAF6FC9DDDA");
+
+var requestOptions = {
+  method: 'DELETE',
+  headers: myHeaders,
```
Comment (Akayeshmantha) [inline source/includes/User/role.md]: ditto

**Review by Akayeshmantha — CHANGES_REQUESTED** [review]
Pls address the changes.

File: source/includes/User/role.md (hunk 1 above)
Comment (dino-saurabh) [reply source/includes/User/role.md]: ok @Akayeshmantha should I make it requestHeader will it be apt?

Comment (dino-saurabh) [reply source/includes/User/role.md]: I will make changes in all the examples if this looks good 👍

Comment (Akayeshmantha) [reply source/includes/User/role.md]: it looks good no need to change on the already  committed once please change on existing PR's

**Comment by dino-saurabh** [comment]
@Akayeshmantha updated with the changes .

**Comment by Akayeshmantha** [comment]
LGTM

**Pull request merged**